src/js/view/controls/controlbar.js

```diff
@@ -277,9 +277,9 @@ export default class Controlbar {
         _model.change('dvrLive', (model, dvrLive) => {
             const { liveBroadcast, notLive } = localization;
             const liveElement = this.elements.live.element();
-
-            toggleClass(liveElement, 'jw-dvr-live', !dvrLive);
-            setAttribute(liveElement, 'aria-label', dvrLive ? liveBroadcast : notLive);
+            const dvrLiveFalse = dvrLive === false;
+            toggleClass(liveElement, 'jw-dvr-live', dvrLiveFalse);
+            setAttribute(liveElement, 'aria-label', !dvrLiveFalse ? liveBroadcast : notLive);
```

Comment:
remove negation for easier readability
`setAttribute(liveElement, 'aria-label', dvrNotLive ? notLive : liveBroadcast);`